exploring_UV_extinction_curves_pt2/Exploring_UV_extinction_curves_pt2.ipynb

```diff
@@ -0,0 +1,746 @@
+{
```

Comment:
<p>I was actually wondering above whether or not it was necessary to continue to treat the lw and sw spectra separately, so I scrolled down to see if we ever combine them. Would it be simpler to just combine them at the end of Part 1? Then there are fewer data files, fewer lines of code, etc.</p><p> </p><p>Likewise, does each array need to be saved in a separate file? There could just be 1 file per object, with two columns, one for wavelength and one for flux. Then they could be loaded in as an astropy table, for example. [I know this is an odd place to put this comment, but it's the place where I realized it.]</p><p> </p><p>Actually - they are supposed to have the same wavelength arrays, right? (Do they?) If so did we need to be carrying all 4 arrays around, or could all the data be saved in one table that is 3 columns long: wavelength, star 1 flux, star 2 flux</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>